services/scan/src/precinct_scanner_interpreter.ts

```diff
@@ -0,0 +1,310 @@
+import { interpret as interpretNh } from '@votingworks/ballot-interpreter-nh';
```

Comment:
I renamed and edited this from `simple_interpreter`, and unfortunately git couldn't keep up to show a good diff. The gist is that I made the interpreter store the config and moved out utility functions that weren't actually about interpretation.